### DISCUSSION TITLE
使用非BMP的PAG素材会有部分图层播放不完整

### DISCUSSION THREAD

**Discussion by yufengvac** [Tencent/libpag#2125] [Q&A] [answered]
计同学输出了一个PAG素材。预期效果是要先出大雪花、再出人物的。 libpag 版本是GitHub master分支最新代码。
在Mac端的PAGViewer上预览是没问题的，但运行在GitHub master分支的libpag客户端上（Android&iOS），前面雪花部分的素材没有展示出来
有问题的素材PAG：
[role.pag.zip](https://github.com/Tencent/libpag/files/14094594/role.pag.zip)
运行视频：
https://github.com/Tencent/libpag/assets/7565523/f65e8794-1eda-40f4-97f9-4382b2f133f4

设计同学转换成 bmp 再次运行了下， 雪花素材就可以完整展示了
转换成 bmp 的素材PAG：
[role_bmp.pag.zip](https://github.com/Tencent/libpag/files/14094647/role_bmp.pag.zip)
运行视频：
https://github.com/Tencent/libpag/assets/7565523/2edfb193-edd9-49db-9b56-7ebe01925004

请问下，这可能是什么问题导致的？
附：设计运用到的能力
<img width="1003" alt="企业微信截图_55bcd6b4-48c9-4b9b-884c-7c8ff28bdd85" src="https://github.com/Tencent/libpag/assets/7565523/f5698ebb-951c-4edf-b010-74213a980062">

**Comment by kevingpqi123** [comment, marked as answer]
<img width="920" alt="image" src="https://github.com/Tencent/libpag/assets/16609221/b1097e17-5087-4ba3-8dfb-79bb3aaa3039">
你为啥不截取上面的内容呢，文档中说的很明白哈，3D 功能社区版没有支持，企业版才会支持，企业版并不是免费使用的，对接方式和收费标准可以在官网上联系我们的工作人员
